src/main/scala/uk/gov/hmrc/ct/computations/calculations/AdjustedTradingProfitOrLossCalculator.scala

```diff
@@ -36,12 +38,20 @@ trait AdjustedTradingProfitOrLossCalculator extends CtTypeConverters {
                                    cp186: CP186,
                                    cp91: CP91,
                                    cp670: CP670,
-                                   cp668: CP668): CP118 = {
-    CP118((profit(cp44, cp54, cp59, cp186, cp91, cp670, cp668) min 0).abs)
+                                   cp668: CP668,
+                                   cpq19: CPQ19,
+                                   cp997c: CP997c): CP118 = {
+    CP118((profit(cp44, cp54, cp59, cp186, cp91, cp670, cp668, cpq19, cp997c) min 0).abs)
   }
 
-  private def profit(cp44: CP44, cp54: CP54, cp59: CP59, cp186: CP186, cp91: CP91, cp670: CP670, cp668: CP668): Int =
-    cp44 + cp54 - cp59 - cp186 + cp91 + cp670 - cp668
+  private def profit(cp44: CP44, cp54: CP54, cp59: CP59, cp186: CP186, cp91: CP91, cp670: CP670, cp668: CP668, cpq19: CPQ19, cp997c: CP997c): Int = {
+    val profit = cp44 + cp54 - cp59 - cp186 + cp91 + cp670 - cp668
```

Comment:
For such places in code it would be nice to have an external reference that could be reviewed be business.